ds121_slides/lecture_2.qmd

```diff
@@ -0,0 +1,809 @@
+---
+title: "Vectors and Linear Combinations"
+author: "CDS DS 121<br>Boston University"
+format: 
+    revealjs:
+        css: 
+        - styles.css
+---
+
+## Learning Objectives
+<!-- Is it better to increment bullets one by one? Or is it better to reveal them together so it is easier to present? -->
+::::{style="font-size: .8em"}
+Today we will talk about vectors.
+
+We will discuss why we need vectors in the first place. We will look at vectors from three perspectives:
+
+:::{.fragment}
+- Algebraic,
+- Geometric, and
+- Computational.
+:::
+
+:::{.fragment}
+_In addition_, we will discuss how to compare and combine vectors.
+:::
+::::
+## Why Vectors?
+
+:::::{.columns}
+::::{.column width=50%"}
+![](/images/lecture_2/oranges_apples.jpg)
+
+:::{.center-text}
+"You **can't** add _apples_ and _oranges_"
+:::
+::::
+::::{.column width="50%"}
+
+:::{style="font-size: 1.2em;"}
+
+_Two_ seperate numbers $v_1$ and $v_2$ form a _**two-dimensional**_ vector:
+
+:::{.center-text}
+
+:::{style="font-size: 1.25em;"}
+$$
+\begin{bmatrix} \mathit{v_1} \\ \mathit{v_2} \end{bmatrix}
+$$
+:::
+:::
+:::
+::::
+:::::
+
+<!-- :::{.text-center}
+You can't add apples and oranges
+::: -->
+
+## Algebraic Representation
+::::{style="font-size: .8em"}
+```{python}
+from IPython.core.display import HTML
+
+def generate_html():
+    return """
+    <div class="purple-box">
+        <p>A list of real numbers with only one column is called a <b>column vector</b>, or simply a <b>vector</b>.
+        </p>
+    </div>
+    """
+
+html_content = generate_html()
+display(HTML(html_content))
+```
+
+<!-- Fun way to get a half line break : ) -->
+<div style="margin-bottom: 0.5em;"> </div> 
+
+:::{.fragment}
+```{python}
+from IPython.core.display import HTML
+
+def generate_html():
+    return r"""
+    <div class="purple-box">
+        <p>The set of all vectors with \( d \) is denoted by \( \mathbb{R^2}\).
```

Comment:
This should be "The set of all vectors with d entries is denoted by R^d." So the word 'entries' is missing, and the dimension should be d, not 2.